app/assets/stylesheets/ghetto_styles.css.scss

```diff
@@ -0,0 +1,1500 @@
+  text-align: center;
+  height: 100%;
+  width: 100%;
+}
+
+/* line 266, ../../app/assets/stylesheets/new_styles/_base.scss */
+#top-right-nav {
+  z-index: 10;
+  position: absolute;
+  right: 10px;
+  top: 10px;
+}
+/* line 272, ../../app/assets/stylesheets/new_styles/_base.scss */
+#top-right-nav > a {
+  -webkit-transition: opacity 0.1s linear;
+  -moz-transition: opacity 0.1s linear;
+  -o-transition: opacity 0.1s linear;
+  transition: opacity 0.1s linear;
+  filter: alpha(opacity=40);
+  -moz-opacity: 0.4;
+  -khtml-opacity: 0.4;
+  opacity: 0.4;
+  margin-left: 5px;
+}
+/* line 278, ../../app/assets/stylesheets/new_styles/_base.scss */
+#top-right-nav > a:hover {
+  filter: alpha(opacity=75);
+  -moz-opacity: 0.75;
+  -khtml-opacity: 0.75;
+  opacity: 0.75;
+  text-decoration: none;
+}
+
+/* line 285, ../../app/assets/stylesheets/new_styles/_base.scss */
+#header {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  padding: 15px;
+  z-index: 30;
+  -webkit-box-shadow: inset 0 -10px 10px -8px rgba(0, 0, 0, 0.8);
+  -moz-box-shadow: inset 0 -10px 10px -8px rgba(0, 0, 0, 0.8);
+  border-bottom: 1px solid #333333;
+  color: #fff;
+  background-color: #333;
+  background-size: cover;
+  background-attachment: fixed;
+}
+/* line 304, ../../app/assets/stylesheets/new_styles/_base.scss */
+#header h1 {
+  filter: alpha(opacity=40);
+  -moz-opacity: 0.4;
+  -khtml-opacity: 0.4;
+  opacity: 0.4;
+}
+
+/* bootstrap label fixes for Roboto */
+/* line 310, ../../app/assets/stylesheets/new_styles/_base.scss */
+.label {
+  padding: 2px 5px;
+  padding-bottom: 3px;
+}
+/* line 314, ../../app/assets/stylesheets/new_styles/_base.scss */
+.label span {
+  display: inline-block;
+  position: relative;
+  top: 1px;
+  font-family: Roboto-Bold;
+}
+
+
+
+/* font overrides */
+/* Roboto */
+@font-face {
+  font-family: Roboto;
+  src: url(/assets/fonts/Roboto-Regular.ttf);
+  weight: normal;
+}
+
+@font-face {
+  font-family: Roboto-Bold;
+  src: url(/assets/fonts/Roboto-Bold.ttf);
+  weight: normal;
+}
+
+@font-face {
+  font-family: Roboto-Light;
+  src: url(/assets/fonts/Roboto-Light.ttf);
+  weight: normal;
+}
+
+/* line 20, ../../app/assets/stylesheets/new_styles/_typography.scss */
+body, p, h1, h2, h3, h4, h5, h6, textarea, input, * {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: normal;
+}
+
+/* line 25, ../../app/assets/stylesheets/new_styles/_typography.scss */
+a {
+  font-family: inherit;
+}
+
+/* login */
+/* line 1, ../../app/assets/stylesheets/new_styles/_login.scss */
+#login {
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding-top: 200px;
+  min-width: 100%;
+  text-align: center;
+}
+/* line 12, ../../app/assets/stylesheets/new_styles/_login.scss */
+#login input[type=text],
+#login input[type=password] {
+  width: 120px;
+}
+/* line 16, ../../app/assets/stylesheets/new_styles/_login.scss */
+#login #huge-text {
+  font-family: Roboto-Light;
+  font-size: 200px;
+  color: #ddd;
+  text-shadow: 0 1px 0 white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.